src/snowflake/cli/plugins/git/manager.py

```diff
@@ -27,3 +30,21 @@ def create(
         if secret is not None:
             query += f"git_credentials = {secret}\n"
         return self._execute_query(query)
+
+    def execute(
+        self,
+        repository_path: str,
+        on_error: OnErrorType,
+        variables: Optional[List[str]] = None,
+    ):
+        return super().execute(
+            stage_path=repository_path, on_error=on_error, variables=variables
+        )
+
+    @staticmethod
+    def get_stage_name_from_path(path: str, is_git_repository: bool = False):
```

Comment:
Where do we use it?

Reply:
https://github.com/snowflakedb/snowflake-cli/pull/970/files#diff-413a7c7aa6000c534346764978e1831f392a3ee947fe572b93b228a71ee0f545L229

`is_git_repository` isn't needed